### PR TITLE
corrected import sentence

### PR DESCRIPTION
**Pull request by iharshit009**

### CODE REVIEW

**Review by sahilit — DISMISSED** [review]

**Comment by narender2031** [comment]
@iharshit009 Why are you adding `..` before the import? what kind of issue are you facing? 
Also please change the folder name`languageBase` to `translations`.

File: src/i18n.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-import Bengali from '@/languageBase/be.json';
-import Urdu from '@/languageBase/ur.json';
-import Assamese from '@/languageBase/as.json';
-import Gujarati from '@/languageBase/gu.json';
-import Kannada from '@/languageBase/kn.json';
-import Kashmiri from '@/languageBase/ks.json';
-import Konkani from '@/languageBase/ko.json';
-import Maithili from '@/languageBase/ma.json';
-import Malayalam from '@/languageBase/ml.json';
+import English from '../languageBase/en.json';
```
Comment (narender2031) [inline src/i18n.js]: change the folder name `languageBase` to `translations`

**Review by narender2031 — CHANGES_REQUESTED** [review]

**Comment by iharshit009** [comment]
> @iharshit009 Why are you adding `..` before the import? what kind of issue are you facing?
> Also please change the folder name`languageBase` to `translations`.

On adding "@" this could not recognize and disables all the imports

File: src/i18n.js (hunk 1 above)
Comment (iharshit009) [reply src/i18n.js]: resolved

**Comment by iharshit009** [comment]
@sarthology @sahilit @narender2031 Made the changes

File: src/i18n.js [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-import Bengali from '@/languageBase/be.json';
-import Urdu from '@/languageBase/ur.json';
-import Assamese from '@/languageBase/as.json';
-import Gujarati from '@/languageBase/gu.json';
-import Kannada from '@/languageBase/kn.json';
-import Kashmiri from '@/languageBase/ks.json';
-import Konkani from '@/languageBase/ko.json';
-import Maithili from '@/languageBase/ma.json';
-import Malayalam from '@/languageBase/ml.json';
+import English from '../translations/en.json';
```
Comment (narender2031) [inline src/i18n.js:3]: why do you need `../`? Are you facing an error with `@/`? if yes? then what is the error?

**Review by narender2031 — CHANGES_REQUESTED** [review]

**Comment by iharshit009** [comment]
When I use @ it is not able to find the file at the location...

**Comment by narender2031** [comment]
ok. I will check this later, for now, I will approve

**Review by narender2031 — DISMISSED** [review]

**Review by sahilit — APPROVED** [review]

**Review by narender2031 — APPROVED** [review]

**Pull request merged**